feature/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go

```diff
@@ -200,6 +214,14 @@ func TestLinkQualification(t *testing.T) {
 	t.Logf("dut1: %v, dut2: %v", dut1.Name(), dut2.Name())
 	t.Logf("dut1 dp1 name: %v, dut2 dp2 name : %v", dp1.Name(), dp2.Name())
 
+	dut_list := []*ondatra.DUTDevice{dut1, dut2}
+	for _, dut := range dut_list {
```

Comment:
Go style is camelCase, not snake_case. In this case, I would just inline the dut list:
`for _, dut := range []*ondatra.DUTDevice{dut1, dut2} {`

Reply:
done. thanks.